exercise-solutions/quiz-game/part-01/code.js

```diff
@@ -37,7 +37,7 @@ function answerTag(prefix, answer, correct) {
         onclick = "alert('Wrong answer');";
     }
 
-    const html = "<div class='gameBtn' onclick=\""+onclick+"\">" + prefix + answer + "</div>";
+    const html = "<div class='answers' onclick=\""+onclick+"\">" + prefix + answer + "</div>";
```

Comment:
why plural `s`?

Reply:
good question. Your correct should be answer. Thanks!

Reply:
fixed